### PR TITLE
Update hip-227 response JSON

### PR DESCRIPTION
**Pull request by Nana-EC**
Signed-off-by: Nana-EC <nana.essilfie-conduah@hedera.com>

**Description**:
Update Contract Results log response `/api/v1/contracts/{id}/results/logs` with missing values

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-227.md [hunk 1]
```diff
@@ -71,7 +74,6 @@ Optional filters
 - `limit`
 - `order`
 - `timestamp`
```
Comment (steven-sheehy) [inline HIP/hip-227.md:76]: We should document what values are supported like the comparison operators or if it's hex-encoded and `0x` is optional, etc.

File: HIP/hip-227.md [hunk 2]
```diff
@@ -71,7 +74,6 @@ Optional filters
...
-- `address`
 - `topic0`
```
Comment (steven-sheehy) [inline HIP/hip-227.md:77]: We should mention able the idiosyncrasies around this API including that timestamp is required if a topic is supplied. Also that timestamp must have a to and from within 7 days of each other. And that there's no paging so users will need to adjust their parameters on their own if more than page size results.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: HIP/hip-227.md (hunk 1 above)
Comment (Nana-EC) [reply HIP/hip-227.md]: Added

File: HIP/hip-227.md (hunk 2 above)
Comment (Nana-EC) [reply HIP/hip-227.md]: Noted, quite a few for this endpoint

File: HIP/hip-227.md [hunk 3]
```diff
@@ -52,30 +55,32 @@ The following JSON represents a typical response result from either of these cal
...
 
-- `limit`
-- `order`
-- `timestamp`
-- `address`
-- `topic0`
-- `topic1`
-- `topic2`
-- `topic3`
+- `index`: Contract log index
```
Comment (steven-sheehy) [inline HIP/hip-227.md:74]:
```suggestion
- `index`: The unique log index within a particular contract result. Starts at zero and increases.
```

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: HIP/hip-227.md [hunk 4]
```diff
@@ -52,30 +55,32 @@ The following JSON represents a typical response result from either of these cal
...
+- `limit`: The maximimum number of items to return in response.
+- `order`: The order in which items are listed - asc (default) or desc.
```
[the remark this answers is not in the document]
Comment (Nana-EC) [reply HIP/hip-227.md]: Updated

File: HIP/hip-227.md [hunk 5]
```diff
@@ -52,30 +55,32 @@ The following JSON represents a typical response result from either of these cal
...
+- `timestamp`: The consensus timestamp in seconds.nanoseconds format. Required when a topic is specified and should not span a time range of more than seven days.  The following optional comparison operators can be added - :gt(e), :lt(e), :eq.
```
[the remark this answers is not in the document]
Comment (Nana-EC) [reply HIP/hip-227.md]: Updated

File: HIP/hip-227.md (hunk 3 above)
Comment (Nana-EC) [reply HIP/hip-227.md]: Updated

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**